src/document.h

```diff
@@ -972,10 +972,13 @@ struct Document {
         } else if (uk >= ' ') {
             if (!selected.g) return NoSel();
             Cell *c = selected.ThinExpand(this);
-            if (!c) return OneCell();
+            if (!c) {
+                selected.Wrap(this);

```

Comment:
Does this always result in a single cell?

Reply:
In `Selection::Wrap(Document*)` there is `xs = ys = 1` so I assume yes.